docs/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ Version 3.2?.? (February 1, 2022)
 The `wpi.sh` script supports non-standard names for build system compile targets
 via the new `-c` command-line option.
 
+The Checker Framework now more precisely computes and checks the type of the
```

Comment:
What should the version be for this release?

Reply:
I suggest 3.21.2, but I'm open to other numbers.

Reply:
That's what I was thinking.